index.html

```diff
@@ -78,93 +124,64 @@ <h1>I'm Tarikwa <br> Glad to see you!</h1>
+            </ul>
+                <ul class="outer-item">
+                    <li class="programming"><img src="./images/javaScript.png" alt="JavaScript"></li>
+                    <li><p>JavaScript</p></li>
                 </ul>
-                    <ul class="outer-item">
-                        <li class="programming"><img src="./images/javaScript.png" alt="JavaScript"></li>
-                        <li><p>JavaScript</p></li>
-                    </ul>
-                    <ul class="outer-item">
-                        <li class="programming"><img src="./images/html.png" alt="HTML"></li>
-                        <li><p>HTML</p></li>
-                    </ul>
-                    <ul class="outer-item">
-                        <li class="programming"><img src="./images/css.png" alt="CSS"></li>
-                        <li><p>CSS</p></li>
-                    </ul>
-                <ul class="framework">
-                    <li class="flex-items"><p>Frameworks</p></li>
-                    <li><i class="fa fa-chevron-right" aria-hidden="true"></i></li>
+                <ul class="outer-item">
+                    <li class="programming"><img src="./images/html.png" alt="HTML"></li>
+                    <li><p>HTML</p></li>
                 </ul>
-                <ul class="skills">
-                    <li class="flex-items"><p>Skills</p></li>
-                    <li><i class="fa fa-chevron-right" aria-hidden="true"></i></li>
+                <ul class="outer-item">
+                    <li class="programming"><img src="./images/css.png" alt="CSS"></li>
+                    <li><p>CSS</p></li>
                 </ul>
-            </section>
-        </section>
-    </header>
+            <ul class="framework">
+                <li class="flex-items"><p>Frameworks</p></li>
+                <li><i class="fa fa-chevron-right" aria-hidden="true"></i></li>
+            </ul>
+            <ul class="skills">
+                <li class="flex-items"><p>Skills</p></li>
+                <li><i class="fa fa-chevron-right" aria-hidden="true"></i></li>
+            </ul>
+      </section>
+        <section class="contact">
+          <div class="inner-div">
+            <h2>Contact me</h2>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+          </div>
+          <form action="https://formspree.io/f/xbjwrzvy" class="form" method="post">
+            <ul class="inner-list">
+              <li><input type="text" placeholder="Enter your name" name="name"  id="name" size="40" maxlength="30"></li>
+              <li><input type="email" name="email" id="email" placeholder="alexandra@gmail.com" pattern="./@gmail.com" required></li>
+              <li><textarea name="text" id="text" placeholder="Write your message here" required></textarea></li>
+              <li><button type="submit">Get It Touch</button></li>
```

Comment:
- [ ] I this it would be nice if you add styling interactions to this button. you can follow the template design
![Screenshot from 2021-11-30 21-48-21](https://user-images.githubusercontent.com/65068771/144117387-32a24bd6-1178-4175-99c5-5352ccde98ab.png)

Reply:
I have implemented the button effects as well.